python/cudf_kafka/setup.cfg

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = cudf_kafka/_version.py
+versionfile_build = cudf_kafka/_version.py
+tag_prefix = v
+parentdir_prefix = cudf_kafka-
+
+[isort]
+line_length=79
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+order_by_type=True
+known_dask=
+    dask
+    distributed
+    dask_cuda
+known_rapids=
+    rmm
+    cudf
+    cuml
+    cugraph
+    dask_cudf
+known_first_party=
+    streamz
```

Comment:
May want to move this into the `known_dask` section and add `cudf_kafka` to the `known_first_party` section

Reply:
I wasn't 100% sure what you mean't by "this" let me know if the change is not what you meant. I'm basically a newbie with setup.cfg files

Reply:
Yea I just meant to change `streamz` --> `cudf_kafka` and then add `streamz` to the `known_dask` section.